portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * Translates text.
+ */
+
+function requestTranslation() {
+        const text = document.getElementById('text').value;
```

Comment:
nit: This looks like it's tabbed twice over instead of once

General statement: Use the autoformatter so you don't have to worry about small nits like this